### DISCUSSION TITLE
Everything is good but I'm still rejected

### DISCUSSION THREAD

**Discussion by HappySunnySun** [community/community#141090] [GitHub Education]
### Hi everyone,

Hello !

I want to apply for Github Education, but everytime I apply, I get rejected while I check all the options to qualify for it.
It's maybe because I use IvCam to take the picture of my document (it have a "IvCam" watermark on it)  (my webcam quality is very low and my phone is not much better)
It's litteraly my 14th request, I'm getting desesperate
Already thanks 4 ur help 

- Noah

PS : I just tried now with my webcam and I still get refused...

**Comment by kelvin-owino** [comment]
Me too #2

**Comment by jekabso21** [comment]
I was verified as a student for 2 years. I asked my school to provide a document confirming that I am still enrolled for 1 more year, including my first and last name, but I am still being denied.

**Reply by HappySunnySun** [reply]
Can someone please help me ? I really need my GitHub Education

**Reply by DBryja** [reply]
Same problem here
